src/tests/TestPasswordGenerator/TestPasswordGenerator.cpp

```diff
@@ -0,0 +1,32 @@
+// Copyright 2005-2017 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#include <QtCore>
+#include <QtTest>
+
+#include "PasswordGenerator.h"
+
+// Get the password alphabet from PasswordGenerator.
+extern QVector<QChar> mumble_password_generator_alphabet();
```

Comment:
Nice trick to work around public visibility in program code but be usable in this test. :)